### ISSUE TITLE
Permanent non-interactive login

### ISSUE THREAD

**Issue by soulofmischief** [keybase/client#16786]
Oneshot mode is useful, however it does not satisfy my use case.

I use Ansible to provision a suite of virtual machines on my desktop and on a couple of servers. I use keybase repos extensively. Even when the ansible playbook isn't running, I would like to remain logged into a read-only account to facilitate automated synchronization. Right now, I have to ssh into a new machine to login interactively, as `expect` is not acceptable in a production environment as each new machine modifies the numerical IDs.

This is exacerbated by the fact that it is a readonly account. This means that I *must* also ssh into a machine which is always logged into the readonly account and only exists for that single purpose, just to provision a device. In practice, this means logging out of my non-readonly account on one machine, logging back in as readonly, provisioning the new machine, logging out, and logging back in as the non-readonly account. This is a major headache.

A simple `--paperkey` option for `keybase login` should suffice.

**Comment by maxtaco** [comment]
@soulofmischief I couldn't follow exactly, will these keybase devices retain storage across runs, or will they be constantly be reprovisioning and regenerating keys?

**Comment by soulofmischief** [comment]
Yes, they will retain storage. Some of these are workspaces which I use as daily drivers and I also make use of kbfs to share files. :)

**Comment by maxtaco** [comment]
got it;  we can make that work, it wouldn't be too much work. probably the same thing where paper key is via stdin by default, so it's not visible to other processes.

**Comment by soulofmischief** [comment]
Awesome. Thanks a ton!

stdin is even better. I guess a flag still wouldn't hurt for those who need it.

**Comment by soulofmischief** [comment]
Would it be a good idea to also include the name of the new machine as a flag to make logging in and provisioning a one liner?
